app/assets/javascripts/table-select-all.es6

```diff
@@ -36,7 +44,8 @@ $(document).ready(function() {
     });
 
     $('table input').click(function() {
```

Comment:
This code adds a listener to every input in _every_ table on the page. As the table where this listener is operating is already identified with the variable `table` this would be better:
```javascript
table.find('input').click(function() {
```